lib/syscontrol/syscontrol.go

```diff
@@ -106,6 +108,14 @@ func GetQuerySeriesLimit() int {
 	return querySeriesLimit
 }
 
+func SetTimeFilterProtection(swithon bool) {
```

Comment:
it is advised to name the "switch on" to "switch" or "enable" variable.

Reply:
> It is advised to name the "switchon" to "enable" variable.

yes